frontend/packages/shared/src/components/altinnHeader/AltinnHeader.test.tsx

```diff
@@ -3,14 +3,19 @@ import { render as rtlRender, screen } from '@testing-library/react';
 import { AltinnHeader, AltinnHeaderProps } from './AltinnHeader';
 import { Button } from '@digdir/design-system-react';
 import { textMock } from '../../../../../testing/mocks/i18nMock';
+import { RepositoryType } from 'app-shared/types/global';
+import { RoutePaths } from 'app-development/enums/RoutePaths';
+import { TopBarMenu } from 'app-shared/enums/TopBarMenu';
+import { MemoryRouter } from 'react-router-dom';
 
 describe('AltinnHeader', () => {
   it('should render AltinnHeaderMenu', () => {
     render({
-      menu: [
+      menuItems: [
         {
-          key: 'test-key',
-          link: <a href='somewhere'>test-key</a>,
+          key: TopBarMenu.About,
+          link: RoutePaths.Overview,
```

Comment:
should we just hard-code the link value here? I think thats better than importing enums from app-development, since shared should not know about app-development app. 😊